File: playground/inspire12/performance-testing-architecture/spring-order/src/main/java/com/order/perf/domain/Delivery.java
package com.order.perf.domain;

import com.order.perf.common.BaseEntity;
import lombok.Getter;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Getter
@Table(name = "DELIVERYS")
@Entity
public class Delivery extends BaseEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "delivery_id", nullable = false)
    private Long id;

    @Column(name = "recipient_name", nullable = false)
    private String recipientName; // 받으실 분

    @Column(name = "mobile", nullable = false)
    private String mobile; // 연락처

    @Column(name = "address", nullable = false)
    private String address; // 배송 주소

    @Column(name = "zip_code", nullable = false)
    private String zipCode; // 우편 번호

    @Column(name = "store_password", nullable = false)
    private String storePassword; // 가게 비밀 번호

    @Column(name = "delivery_memo", nullable = false)
    private String deliveryMemo; // 배송 메모

    protected Delivery() {
    }

    public Delivery(final String recipientName, final String mobile,
                    final String address, final String zipCode,
                    final String storePassword,
                    final String deliveryMemo) {
        this.recipientName = recipientName;
        this.mobile = mobile;
        this.address = address;
        this.zipCode = zipCode;
        this.storePassword = storePassword;
        this.deliveryMemo = deliveryMemo;
    }
}

File: playground/ruthetum/app/e-commerce/order/src/main/kotlin/team/sipe/spurt/ecommerce/order/domain/Delivery.kt
package team.sipe.spurt.ecommerce.order.domain

import jakarta.persistence.*
import org.hibernate.annotations.Comment

@Table(name = "deliveries")
@Entity
class Delivery(
    @Comment("받으실 분")
    @Column(name = "recipient_name")
    val recipientName: String,

    @Comment("연락처")
    @Column(name = "mobile")
    val mobile: String,

    @Comment("배송 주소")
    @Column(name = "address")
    val address: String,

    @Comment("우편 번호")
    @Column(name = "zip_code")
    val zipCode: String,

    @Comment("가게 비밀 번호")
    @Column(name = "store_password")
    val storePassword: String,

    @Comment("배송 메모")
    @Column(name = "delivery_memo")
    val deliveryMemo: String,

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null,
): team.sipe.spurt.ecommerce.order.common.BaseEntity() {
}

